docs/tutorials/IndexerMigration.md

```diff
@@ -126,3 +122,145 @@ graph LR;
+
+
+## Altering Architecture of Indexer Applications
+
+Let's take a look at a theoretical application more closely and see what in particular we need to do to alter it to use
+Conduit. 
+
+Here's a diagram of a containerized application (leaving out some details such as load balancers, and scaled out API
+nodes).
+
+```mermaid
+graph LR;
+    algod["Archival Alogd"]
+    indexer["Indexer"]
+    psql["Postgresql"]
+    subgraph "Persistent Volume"
+        ll["Local Ledger"];
+    end
+    
+    indexer-->algod;
+    indexer-->psql;
+    indexer-->ll;
+```
+
+### Step 1: Convert from Archival to Follower Node
+In our hypothetical existing architecture, we likely have a persistent volume storing the algod data directory for our
+Archival node. It is probably still a good idea to maintain the algod's data in a persistent volume, though the amount
+of storage required will be minimal in comparison.
+
+
+If you have an existing Archival node which is on the same round as your Indexer, you can simply stop algod, alter your
+config as follows, and restart it.
+```yaml
+Archival: false,
+MaxAcctLookback: 256,
+CatchupParallelBlocks: 256,
+EnableFollowMode: true
+```
+Your node will then drop all but the last 1000 rounds from your database, and will have `2 * MaxAcctLookback`
+rounds in its cache.
+
+However, if your algod node's round is too far beyond your Indexer's you will need to catchup your node from scratch.
+Follower nodes provide the same methods of catchup as regular algod nodes, but will have the sync round set to the most
+recent ledger round upon startup.
+
+In order to manually catchup your node from 0 to 25 million, for example, you will need to call the sync round API to
+update the ledger constraint:
+```
+curl -X POST -H "X-Algo-API-TOKEN:$ALGOD_TOKEN" http://$ALGOD_ADDR/v2/ledger/sync/25000000
+```
+
+Now you can run fast catchup on your node to the closest catchpoint prior to the desired sync round.
+```
+goal node -d $ALGOD_DIR catchup 25000000#EOX5UYQV4IXTGYQCIDR7ZLUK6WZGDC5EG6PYQGBG6PBYNAQUPN2Q
+```
+Once catchup is complete we will resume normal catchup. If your catchpoint is not exactly the round your database is on,
+you will need to wait for normal catchup to get your node to the desired point in time. The sync round we set earlier will ensure that the node does not
+advance past the round Conduit needs before we're ready. 
+
+### Step 2: Remove the Local Ledger
+Because our Conduit pipeline will use the Follower node's state delta API, we no longer need our local ledger persistent 
+volume. Simply remove it.
+
+### Step 3: Refactor our Indexer Writer to Conduit
+You're free to capture any data you like using Conduit. I'd encourage you to take a look at the filter processor and see
+if you can reduce the amount of data you store in your database by removing non-relevant data.
+
+In order to have parity with the previous Indexer, which stored all data, you can use the following Conduit configuration.
```

Comment:
What do you think about suggesting `./cmd/conduit/conduit init --importer algod --exporter postgresql --data new_data_dir` instead of copy/pasting the config?